dev/deps/spark-deps-hadoop-2.2

```diff
@@ -18,8 +18,8 @@ avro-ipc-1.7.7-tests.jar
 avro-ipc-1.7.7.jar
 avro-mapred-1.7.7-hadoop2.jar
 bonecp-0.8.0.RELEASE.jar
-breeze-macros_2.10-0.11.2.jar
-breeze_2.10-0.11.2.jar
+breeze-macros_2.11-0.12.jar
```

Comment:
we cannot switch to scala 2_11 I think (see #scala Slack channel for talk initiated by Sergeï)